source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs

```diff
@@ -161,6 +169,32 @@ public bool PreviouslyOwnedByPlayer
 			set => Function.Call(Hash.SET_VEHICLE_HAS_BEEN_OWNED_BY_PLAYER, Handle, value);
 		}
 
+		/// <summary>
+		/// Gets a value indicating whether this <see cref="Vehicle"/> allows <see cref="Ped"/>s to rappel.
+		/// </summary>
+		/// <value>
+		/// <c>true</c> if this <see cref="Vehicle"/> allows <see cref="Ped"/>s to rappel; otherwise, <c>false</c>.
+		/// </value>
+		public bool AllowsRappel => Game.Version >= GameVersion.v1_0_757_2_Steam
```

Comment:
Should be `AllowRappel` to stay consistent with other properties with the same prefix (e.g. in Euphoria helpers).